utils/text/levenshtein.py

```diff
@@ -0,0 +1,39 @@
+# Third Party (PyPI) Imports
```

Comment:
1. Can we rename this file to `algorithms.py` and the function name to `levenshtein_distance`
2. Can we also add `numpy` to `requirements.txt`: https://github.com/hacktoolkit/django-htk/blob/master/requirements.txt; Leave it unpinned
3. Inside of https://github.com/hacktoolkit/django-htk/blob/master/utils/text/__init__.py, can you add:
    ```
    from htk.utils.text.algorithms import levenshtein_distance
    ```
    So that it can be imported as `from htk.utils.text import levenshtein_distance`
4. I think this implementation is potential suboptimal, in that it constructs an `m * n` matrix. Can we use this implementation instead: https://en.wikipedia.org/wiki/Levenshtein_distance#Iterative_with_two_matrix_rows (also update the Wikipedia URL in the docstring)
5. Would prefer to use `deletion_cost`, `insertion_cost`, `substitution_cost` as intermediate variable names, instead of `a`, `b`, `c`

Reply:
1. Renamed file and function
2. Added numpy to requirements.txt
3. Added import to utils/text/__init__.py
4. The results of the algorithm from the link https://en.wikipedia.org/wiki/Levenshtein_distance#Iterative_with_two_matrix_rows is not valid and keeping existing implementation as it is. Will check other optimal algorithms to get the valid distance.
5. Renamed the variables